apps/protocol-frontend/src/hooks/useUserDaoMember.ts

```diff
@@ -0,0 +1,9 @@
+import { useUser } from "../contexts/UserContext";
+
+
+export const useUserDaoMember = (daoId: string): boolean | undefined => {
+  const { userDaos } = useUser();
+  if (userDaos !== null && daoId !== undefined) {
+    return userDaos.map(dao => dao.guild_id).includes(parseInt(daoId));
```

Comment:
This doesn't utilize `useQuery`.

Reply:
Yup, makes sense. I think the reasoning to include in UserContext is solid -- I was on the fence but that was part of my consideration. Since it's not really a hook and it does seem to make more sense in Context.